src/main/kotlin/org/jetbrains/tinygoplugin/runconfig/TinyGoRunStateConfig.kt

```diff
@@ -35,6 +39,11 @@ class TinyGoTestRunningState(env: ExecutionEnvironment, module: Module, configur
     override val hardwareArguments: List<String> = emptyList()
 }
 
+class TinyGoHeapAllocRunningState(env: ExecutionEnvironment, module: Module, configuration: TinyGoRunConfiguration) :
+    TinyGoRunningState(env, module, configuration) {
+    override val additionalParameters: List<String> = listOf("-o", "temp.out", "-print-allocs=.")
```

Comment:
Won't you have the same problem here that you were solving in the first commit?

Reply:
Do you mean the problem with read-only file systems?

Reply:
Yes.

Reply:
The first commit on my side is a change in `CHANGELOG.md`. The commit with mentioned fix must be the latest in this branch and will be applied over this code fragment, won't it?

Reply:
Ah, you're right. I've got confused by the Pull Requests tool window order of commits. Please disregard.